packages/core/test/hotkeys/hotkeysTests.tsx

```diff
@@ -114,19 +114,32 @@ describe("Hotkeys", () => {
         });
 
         it("triggers hotkey dialog with \"?\"", (done) => {
+            // this test takes awhile. the hotkey dialog is supposed to take 100ms
```

Comment:
not only does it slow the test suite down -- it also introduces flakiness because these timeouts are dependent on things like CPU load. I would probably prefer to skip this test until we have test-friendly minimal dialogs